libs/kml/types.hpp

```diff
@@ -301,6 +301,49 @@ struct ColorData
   uint32_t m_rgba = 0;
 };
 
+inline std::string toCssColor(ColorData color)
```

Comment:
Had to add function to convert our internal colors to CSS compatible ones, because most GeoJson apps are Web-based (Umap, Geojson.io, etc).

Reply:
1. ToCssColor
2. Remove inline and move the implementation to types.cpp
3. Would it be easier to have it as a member function of ColorData?

Reply:
Renamed to `ToCssColor` and moved function to `types.cpp`.